lib/content-services/src/lib/tag/tag-node-list.component.scss

```diff
@@ -1,7 +1,31 @@
 .adf-tag-node-list {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    width: inherit;
+
+    &.adf-flex-column {
+        flex-direction: column;
```

Comment:
you can just put that on the parent adf-tag-node-list class? is it's used there by default anyway

Reply:
By default it is set to `row` and `column` is used only at the beginning for calculations and in case of first tag being too wide to fit into one line with button, in other cases `row` should remain